recipe/meta.yaml

```diff
@@ -285,6 +314,68 @@ outputs:
         https://docs.nvidia.com/cuda/cuquantum/license.html
       doc_url: https://docs.nvidia.com/cuda/cuquantum/latest/custatevec/
       dev_url: https://github.com/NVIDIA/cuQuantum
+  
+  - name: cuquantum-python
+    # cuquantum python version label is MAJOR.MINOR.PATCH
+    version: {{ ".".join(sdk_version[:3]) }}
+    build:
+      number: {{ build_num }}
+      skip: true  # [py<310 or py>312]
+      # build cuQuantum Python package is CUDA agnostic, serving both CUDA 11/12
+      # only build in the CUDA 11 pipeline
+      # (note: this would still use cuda_compiler_version to calculate hash)
```

Comment:
Maybe this needs to be fixed upstream, but it doesn't seem like NVCC is used at all to build the python modules in this package. So, it doesn't make sense that the python package depends on CUDA. It should be sufficient to depend on the appropriate version of cuquantum.